Formula/ice.rb

```diff
@@ -33,12 +25,24 @@ def install
       "LMDB_HOME=#{Formula["lmdb"].opt_prefix}",
       "CONFIGS=shared cpp11-shared xcodesdk cpp11-xcodesdk",
       "PLATFORMS=all",
-      # We don't build slice2py, slice2js, slice2rb to prevent clashes with
-      # the translators installed by the PyPI/GEM/npm packages.
-      "SKIP=slice2confluence slice2py slice2rb slice2js",
+      "SKIP=slice2confluence",
       "LANGUAGES=cpp objective-c",
     ]
     system "make", "install", *args
+
+    (libexec/"bin").mkpath
+    %w[slice2py slice2rb slice2js].each do |r|
+      mv bin/r, libexec/"bin"
```

Comment:
Why not keeping them in `bin`, where they are in the user's path, if that is the default installation?

Reply:
In the past they've been an optional install with Homebrew was they conflict with binaries installed with the NPM, GEM, and PyPi packages. Putting them in `libexec/bin` users will at least still be able to use these compilers in scripts, etc.